cli/src/index.ts

```diff
@@ -39,7 +39,7 @@ trafficMirrorAws.command("remove").action(awsTrafficMirrorRemove)
 const trafficMirrorGcp = trafficMirror
   .command("gcp")
   .description("Set up traffic mirroring for GCP")
-trafficMirrorGcp.command("new").action(gcpTrafficMirrorSetup)
+trafficMirrorGcp.command("new").action(gcpTrafficMirrorSetup).option("-f,--force", "Force creation of new instance")
```

Comment:
Is force the right word here? Shouldn't it be like `--new-instance` or something

Reply:
the idea is that it forces creation of a new instance instead of updating existing

Reply:
cool